### DISCUSSION TITLE
Unable to renew my expired domain on my vercel dashboard

### DISCUSSION THREAD

**Discussion by princeyeaboah** [vercel/community#2027] [Help]
### Summary

I'm unable to renewing my expired domain. My previous card had expired before the auto-renewal could happen, and since I replaced the card I have been unable to renew the domain.

There's no button or anything related to such that'll enable me to reactivate my domain. Have anyone experience this or know a solution?

**Comment by swarnava** [comment]
Hi,

What is the domain name?

**Comment by princeyeaboah** [comment]
Vercel has resolved it.
